Tests/SonarScanner.MSBuild.Test/BootstrapperClassTests.cs

```diff
@@ -226,6 +228,7 @@ public void Exe_PostProc_Fails()
             logger.AssertWarningsLogged(0);
             logger.AssertErrorsLogged(1);
             AssertPostProcessorArgs();
+            TelemetryTestUtils.AssertTelemetryContent(logger, TelemetryPath(), """{"dotnetenterprise.s4net.scannerEngine.analysisResult":"failure"}""");
```

Comment:
This doesn't work because the ScannerEngine is posting the telemtry to the server. When we know the engine was successful, we can no longer append telemetry (it just isn't send anymore). The good news is, that the engine itself sends telemetry and hopefully `analysisResult` is part of it. In the best case, we do not need to do anything here.
Anyway: This needs a bit more thoughts, specification and synchronization with Denis and analysis experience squad and should not be part of this sprint.